### ISSUE TITLE
run performance and accessibility tests at build time

### ISSUE THREAD

**Issue by harrisonpim** [harrisonpim/theblackhart.co.uk#60]
- [x] lighthouse in vercel console
- [x] generate a sitemap at `/pages/sitemap.xml.js` https://leerob.io/blog/nextjs-sitemap-robots
- [x] pa11y in github ci using sitemap https://github.com/pa11y/pa11y-ci
